foerder.html

```diff
@@ -226,12 +226,18 @@
 				formDict[pair[0]] = pair[1];
 			}
 
-			var jsonMsg = JSON.stringify(formDict);
```

Comment:
Alright. I'll revert my commit, which rewrites the qrcode data and merge this pull request. Then you can check on msgpack. I'll work on a python tool to read the data from the qr code, sanitize it, display it, optionally modify it and writing it to the database after confirmation.